firstvoices/backend/search/indices/dictionary_entry_document.py

```diff
@@ -21,8 +21,9 @@
 @dictionary_entries.document
 class DictionaryEntryDocument(Document):
     _id = Text()
-    title = Text()
+    title = Text(analyzer="standard", ignore_above=256)
```

Comment:
Does this solve fw-3385?

Reply:
I don't believe this addresses FW-3385. The `ignore_above` flags marks a cutoff point after which elasticsearch will not index/store any characters in the field. To solve FW-3385, we will have to limit the text on the search term going into the elasticsearch. I have added a link to the open issue on FW-3385 as a comment. This flag was present in our nuxeo code, and thus I added it here. We can remove it if required.